db/schema.rb

```diff
@@ -59,6 +59,19 @@
     t.datetime "updated_at",       null: false
   end
 
+  create_table "loans", force: :cascade do |t|
+    t.string   "employee"
+    t.string   "item"
+    t.date     "dateBorrowed"
+    t.date     "dateReturned"
```

Comment:
could even follow the convention that rails has given you below

borrowed_at
returned_at